packages/office-ui-fabric-react/src/components/__snapshots__/Callout.Basic.Example.tsx.shot

```diff
@@ -221,7 +221,6 @@ exports[`Component Examples renders Callout.Basic.Example.tsx correctly 1`] = `
                 border: 0px;
               }
           hidden={true}
-          role="alertdialog"
```

Comment:
This seems like it breaks things, shouldn't this role still be there?

Reply:
If you're not viewing in codeflow, you may only be seeing a narrow view of the snapshot, but 20 lines down, you'll see the redundant one that I've preserved and supplemented with aria-roledescription (if present)

---
In reply to: [288761194](https://github.com/OfficeDev/office-ui-fabric-react/pull/9269#discussion_r288761194) [](ancestors = 288761194)

Reply:
Hmmm, I can see that it is preserved below, I'm just a bit confused as to why this one is getting removed since the basic example file doesn't seem to have been changed.

Reply:
In CalloutContent.base.tsx, line 227, I added a filter to the call to getNativeProps, which is removing `role` from the return value and thus removing it from this element. It was already on the below, which is why that line in the snapshot remains unchanged. This also explains why Pivot.Fabric.Example.tsx.shot was updated, as it also renders callout with my change. And these two example files are the only ones that set the `role` prop in the callout props. Make sense?

---
In reply to: [288769776](https://github.com/OfficeDev/office-ui-fabric-react/pull/9269#discussion_r288769776) [](ancestors = 288769776)